tools/setup/install-sgrep

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+if [ ! -d "/usr/local/bin/sgrep-lint-files" ]
+then
+    tmpdir="$(mktemp -d)"
+    trap 'rm -r "$tmpdir"' EXIT
+    cd "$tmpdir"
+    wget -nv "https://github.com/returntocorp/sgrep/releases/download/v0.4.9b2-ubuntu-18.04/sgrep-0.4.9b2-ubuntu-18.04.tgz"
```

Comment:
Looking at the CI results, this binary doesn’t run in Ubuntu 16.04.

Reply:
Yes we built the binary to work in 18.04 since that's what the vagrant dev environment seems to work on. Should we just add this to the skip list like `gitlint` and `mypy`?

Reply:
We support both Ubuntu 16.04 and 18.04 so the tests need to be able to run on both. Our CI runs `lint --groups=frontend` on 16.04 and `lint --groups=backend` on both.

Reply:
Hmm I see. I'll look into statically linking the dependencies of sgrep. But in the mean time is there a way to add sgrep without blocking on this. We could put back the rule in custom check, skip sgrep checking in CI for now, so that devs can evaluate this tool locally?

Reply:
I expect that if you build for Ubuntu 16.04, the same build would run on Ubuntu 18.04—just not the other way around.

Reply:
Looks like a problem still on sgrep's end. Looking into this.

Reply:
@andersk looks like it is failing a front-end casper test. Should I rebase?

Reply:
I restarted the test for you to see if that’s enough; we have a handful of known-flaky Casper tests. Feel free to rebase at any time if you’re unsure, though—we currently use a rebase workflow anyway.